client/src/App.tsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable react/button-has-type */
+import react, { useState } from 'react';
+import reactLogo from './assets/react.svg';
+import './App.css';
+
+function App(): react.ReactElement {
```

Comment:
Clear the component, remove the type, and convert it to an arrow function.
Use named exports.